src/solvers/dgsem/basis_lobatto_legendre.jl

```diff
@@ -242,6 +242,57 @@ end
 
 @inline polydeg(mortar::LobattoLegendreMortarL2) = nnodes(mortar) - 1
 
+struct LobattoLegendreMortarIDP{RealT <: Real, NNODES, Mortar} <:
+       AbstractMortar{RealT}
+    positivity_variables_cons::Vector{Int}
+    mortar_l2::Mortar
+    mortar_weights::Array{RealT, 3}      # [node (large element), node (small element), small element]
+    mortar_weights_sums::Array{RealT, 2} # [node, left/right/large element]
+end
```

Comment:
What is the best way to implement this such that it is easily adaptable or already ready to use in 3D?
Since the local weights (for the piecewise constant basis functions) only depend on the interval in which both basis functions overlap, it should be possible to use the current structure of `mortar_weights` also in 3D by multiplying two 2D local weights.
Nevertheless, to keep the indices of small elements (`1:2` and `1:4`) and to be overall simpler, I'd prefer to adapt both variables depending on the dimension like:

```suggestion
struct LobattoLegendreMortarIDP{RealT <: Real, NNODES, NDIMS, 2NDIMSM1P1, Mortar} <:
       AbstractMortar{RealT}
    positivity_variables_cons::Vector{Int}
    mortar_l2::Mortar
    mortar_weights::Array{RealT, 2NDIMSM1P1} # [node_i (large), node_j (large), node_i (small), node_j (small), small element] of length `2*(NDIMS-1) + 1`
    mortar_weights_sums::Array{RealT, NDIMS} # [node_i, node_j, position]
end
```

Alternatively, `2NDIMSM1=2*NDIMS -1` is the same and simpler as `2NDIMSM1P1=2*(NDIMS-1)+1` but less understandable IMO.
I'm not sure about the best way here.